### ISSUE TITLE
Can you release your trained model?

### ISSUE THREAD

**Issue by Junfeng-Geo** [BailiangJ/spine-ct-mr-registration#4]
Nice work. However, I found that it's really tough to collect more than 160 paired CT and MRI images as training data.  Thus, I would be greatly appreciated if you could release your trained model.

**Comment by BailiangJ** [comment]
Hi @Junfeng-Geo , thank you for your interest.

Unfortunately, the pretrained model can't be released due to the confidentiality of the dataset.

I would recommend to take a look at [SpineWeb](http://spineweb.digitalimaginggroup.ca/Index.php?n=Main.Datasets#Dataset_6.3A_Intervertebral_Disc_Localization_and_Segmentation_Multi-modality_MRI_Spine_Image_Database) to see if there is useful dataset for you.

Or it would also be a substitute to test the method with inter-subject spine scans e.g. CTs. And with the webtool [Anduin](https://bonescreen.de/), you can easily obtain the vertebrae labels of CT.
